app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/summary/ClaimSummaryDestination.kt

```diff
@@ -103,16 +137,112 @@ private fun ClaimSummaryScreen(
     ) {
       Column(sideSpacingModifier.fillMaxWidth()) {
         for ((left, right) in detailPairs) {
-          Row {
-            Text(text = left, modifier = Modifier.weight(1f))
-            Text(text = right, modifier = Modifier.weight(1f), textAlign = TextAlign.End)
+          HorizontalItemsWithMaximumSpaceTaken(
+            startSlot = {
+              Text(text = left)
+            },
+            endSlot = {
```

Comment:
I see the `modifier = Modifier.weight(1f)` was used in here before too, but since now we are inside `HorizontalItemsWithMaximumSpaceTaken` the two slots already get the right amount of space given to them. I think you can make do with just removing the `modifier = Modifier.weight(1f)` completely here? Do correct me if I am wrong here